4.0/entrypoint.sh

```diff
@@ -42,9 +42,11 @@ echo "ENGINE_BASE_URL=\"https://${OVIRT_FQDN}:${HTTPS_PORT}/ovirt-engine/\"" >>
 echo "SSO_ENGINE_URL=\"https://${OVIRT_FQDN}:${HTTPS_PORT}/ovirt-engine/\"" >> /etc/ovirt-engine/engine.conf.d/999-ovirt-engine.conf
 
 export PGPASSWORD=$POSTGRES_PASSWORD
-psql $POSTGRES_DB -h $POSTGRES_HOST -p $POSTGRES_PORT  -U $POSTGRES_USER -c "UPDATE vdc_options set option_value = '$HOST_ENCRYPT' WHERE option_name = 'SSLEnabled';"
-psql $POSTGRES_DB -h $POSTGRES_HOST -p $POSTGRES_PORT  -U $POSTGRES_USER -c "UPDATE vdc_options set option_value = '$HOST_ENCRYPT' WHERE option_name = 'EncryptHostCommunication';"
-psql $POSTGRES_DB -h $POSTGRES_HOST -p $POSTGRES_PORT  -U $POSTGRES_USER -c "UPDATE vdc_options set option_value = '$HOST_INSTALL' where option_name = 'InstallVds';"
-psql $POSTGRES_DB -h $POSTGRES_HOST -p $POSTGRES_PORT  -U $POSTGRES_USER -c "UPDATE vdc_options set option_value = '$HOST_USE_IDENTIFIER' WHERE option_name = 'UseHostNameIdentifier';"
+
+engine-config -s SSLEnabled='$HOST_ENCRYPT'
+engine-config -s EncryptHostCommunication='$HOST_ENCRYPT'
+engine-config -s InstallVds='$HOST_INSTALL'
+engine-config -s UseHostNameIdentifier='$HOST_USE_IDENTIFIER'
+engine-config -s BlockMigrationOnSwapUsagePercentage=99999
```

Comment:
Why do we experience the swap problem in this setup? I have never seen that before.

Please make it configurable.

We should also consider doing something like suggested in #16, to not have to create dozens of environment variables